components/dashboard/src/App.tsx

```diff
@@ -56,6 +56,21 @@ function App() {
         })();
     }, []);
 
+    useEffect(() => {
```

Comment:
**praise**: Good to see useEffect in action. Cross-posting the [dark mode docs]( https://tailwindcss.com/docs/dark-mode) from Tailwind in case these are helpful here.

Reply:
Thanks! That's indeed the docs I used to introduce the dark mode.